src/morse/builder/abstractcomponent.py

```diff
@@ -104,8 +104,11 @@ def _remove_entries(dict_, robot_list):
     def write_config(robot_list):
         """ Write the 'component_config.py' file with the supplied settings """
         if not 'component_config.py' in bpymorse.get_texts().keys():
+            texts_before = set(bpymorse.get_texts())
             bpymorse.new_text()
-            bpymorse.get_last_text().name = 'component_config.py'
+            texts_after = set(bpymorse.get_texts())
```

Comment:
@nicolaje, if you are creating new functionality about accessing BPY, I think that this code should be inside bpymorse. So, I propose that you move this code related to guarantee getting last created text to a bpymorse function, maybe ```get_last_created_text´'' or something similar. What do you think?

Reply:
@dgerod thanks for cleaning the backlog of pull requests!
I propose that the method "new_text()" returns the newly created text.

Since the only way to retrieve the newly created text is to do a set difference, it would require a method that has a state otherwise, which I don't like here.

What do you think?

Reply:
I agree, your proposal is better than mine.

I don't know what is the current status of unit test but we should add test every time we add code. So, could you check if it is possible to add a unit test for this function to test this new functionality? Thanks.